ddln_imaginator.proto

```diff
@@ -185,7 +186,45 @@ message WorldInfo {
     optional LatLongAlt home_geo = 4;
 }
 
+// Imaginator info message which contains imaginator specific information
+// This information is world agnostic and typically generated upon initialization
+message ImaginatorInfo {
+
+    // Reserved for future values such as version info and so on.
+    reserved 1, 2, 3, 4, 5, 6, 7, 8, 9;
+
+    // Object information message
+    message ObjectInfo {
+
+        // True if object info is customized, false in case information is left undefined, e.g. not specified.
+        // Contact sim@daedalean.ai if object information for a specific object key is required.
+        optional bool has_info = 1;
```

Comment:
we should not have objects without info, this will just confuse users

Reply:
consider this done.